index.html

```diff
@@ -1572,7 +1437,8 @@ <h2>The <dfn>NDEFRecordType</dfn> enum</h2>
     is available. The <dfn>NFCReadingEvent.message</dfn> is an <a>NDEFMessage</a> object.
     <dfn>NFCReadingEventInit</dfn> is used to initialize a new event with a serial number
     and the <a>NDEFMessageInit</a> data via the <dfn>NFCReadingEventInit.message</dfn> member.
-    If <dfn>NFCReadingEventInit.serialNumber</dfn> is <a>not present</a> or is `null`,
+    If <dfn>NFCReadingEventInit.serialNumber</dfn> is
```

Comment:
maybe different PR, but this seems like a very odd place to define something.g 

```suggestion
    If <dfn>NFCReadingEventInit.serialNumber</dfn> is
```

Reply:
Exactly.

Reply:
Files https://github.com/w3c/web-nfc/issues/252.